src/ui.toit

```diff
@@ -0,0 +1,259 @@
+// Copyright (C) 2023 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the package's LICENSE file.
+
+import encoding.json
+
+interface Printer:
+  emit o/any --title/string?=null --header/Map?=null
```

Comment:
I think I'd use 'object' instead of 'o' in general.

Reply:
done.